### PR TITLE
Homework 1 Pull Request

### PR DESCRIPTION
**Pull request by emilyrobb**
Emily Robb

Thought this was a really helpful first assignment in learning the whole GitHub workflow by doing!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
 # hw01
+## 10/04/2020
+***
+
+Hi everyone, my name is Emily Robb and I'm a fourth year in the College. I'm double-majoring in Philosophy and Political Science and am incredibly excited to take this course and learn practical skills that I haven't really had previous exposure to. Looking forward to learning with all of you!
+
+![Picture of me](IMG_5373.jpeg)
```
Comment (deblnia) [inline README.md:7]: Nice relative link!

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |     X      |              |            |
| **Reproducibility**         |    X       |              |            |

Remarks:

* Overall, really solid work! Glad the videos were helpful in re: git workflow, and your markdown syntax looks great. :smile:
* Going forward, put your reflection in the pull request!

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+
+---
+
+##### Process Description
+I started this homework by reviewing the overview and instructions, and ultimately watching the accompanying video.
+
+[Panopto Instructions](https://uchicago.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=7bd284fa-ceac-400b-a8db-ab9500f8d943&start=199.610526)
```
Comment (IshitaSinghal98) [inline README.md:14]: To make the markdown more seamless, you can actually hyperlink the video instruction link by simply making the 'Panopto instructions' segment a part of your sentence!

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
 # hw01
+## 10/04/2020
+***
+
+Hi everyone, my name is Emily Robb and I'm a fourth year in the College. I'm double-majoring in Philosophy and Political Science and am incredibly excited to take this course and learn practical skills that I haven't really had previous exposure to. Looking forward to learning with all of you!
```
Comment (IshitaSinghal98) [inline README.md:5]: Your assignment is great - the only thing I'd add is maybe more information about _you_ (Adding your interests, where you're from etc.)!

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+
+2. Copied the link to my clipboard
+
+3. Opened RStudio and created a new *version control* project using the repository URL copied in the last step
+
+To finish the rest of this homework up, I clicked on the README.md file and started writing in it with Markdown, adding different stylistic elements, and even uploading an image.
+
+I really liked this homework assignment and the overall simplicity of Markdown. The GitHub workflow was definitely pretty clunky and confusing at first but I think it will become more natural with more practice over time!
+
+---
```
Comment (IshitaSinghal98) [inline README.md:30]: I like that you've added line dividers, it's a neat way of sectioning of the different segments

**Pull request closed without merging**